src/framework/audio/internal/platform/win/wincoreaudiodriver.cpp

```diff
@@ -27,6 +27,9 @@
 #include "windows.h"
 #include "audioclient.h"
 
+#if defined __MINGW32__ // hack against "undefined reference" when linking in MinGW
+#define INITGUID
+#endif
```

Comment:
If this solution works, then I think it's better

Reply:
It does. The other is commented out, but would work too

Reply:
I suggest keep this, and remove the second one (remove comments)

Reply:
OK, gimmi some secs...

Reply:
done